src/cloudforet/setup.py

```diff
@@ -0,0 +1,37 @@
+#
+#   Copyright 2022 The Cloudforet Authors.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from setuptools import setup, find_packages
+
+with open('VERSION', 'r') as f:
+    VERSION = f.read().strip()
+    f.close()
+
+setup(
+    name='cloudforet-console-api-v2',
+    version=VERSION,
+    description='Cloudforet console-api-v2 service',
+    long_description='',
+    url='https://www.cloudforet.dev/',
+    author='MEGAZONE CloudOne Team',
+    author_email='admin@cloudforet.dev',
```

Comment:
Change author and author_email:
- author: Cloudforet Admin
- author_email: admin@cloudforet.io